app/Models/AccessPoint.php

```diff
@@ -26,6 +27,6 @@ class AccessPoint extends DeviceRelatedModel implements Keyable
 
     public function getCompositeKey()
     {
-        return "{$this->name}_{$this->radio_number}";
+        return "{$this->device_id}_{$this->mac_addr}_{$this->radio_number}";
```

Comment:
It doesn't seem like you would need both mac_addr and radio_number

Reply:
Are you sure? Each physical AP has an unique MAC but can have multiple radios configured (== a/b/g/n...) so the same MAC has multiple records. Each radio can also have different data such as client counts. To uniquely identify each record, I think both of them are needed. However, using the MAC is sufficient for uniquely identifying each physical device.